v2/tools/generator/internal/astmodel/property_reference.go

```diff
@@ -43,6 +43,6 @@ func (ref PropertyReference) IsEmpty() bool {
 
 // String returns a string representation of this property reference
 func (ref PropertyReference) String() string {
-	g, v, _ := ref.declaringType.PackageReference.GroupVersion()
+	g, v := ref.declaringType.PackageReference.GroupVersion()
```

Comment:
Not sure if we should use `TryGroupVersion()` here? Or would `PackageReference` always be local reference?

Reply:
A `PropertyReference` is a reference to a property on one of the types we decare - which are never in external packages, so the reference will always be either local or storage.